src/Views/Movie/EditMovie.cshtml

```diff
@@ -5,11 +5,16 @@
 <h2>Edit movie</h2>
 
 @Html.Partial("_ShowMovie")
-<form asp-action="AddComment" asp-controller="Movie">
-    <textarea class="form-control" 
-            cols="20" 
-            id="comment"
-            name="comment"
-            rows="2">Tell us what you thought...</textarea>
-    <input type="submit" value="Send" />
-</form>
+<textarea id="userComment" 
+        cols="20"
+        name="comment"
+        rows="2">Tell us what you thought...</textarea>
```

Comment:
Just as an extra "html practice", try using a placeholder instead of setting the value here